src/topics/group-generator/group-generator.ts

```diff
@@ -284,6 +299,20 @@ export class GroupGeneratorService {
             : ({
                 accountsNumber: Object.keys(group.data).length,
                 valueDistribution: { 1: Object.keys(group.data).length },
+                minValue: Object.values(group.data)
```

Comment:
all the logic should be in "computeProperties". Filter "group.name === "sismo-contributors"" there